{"name": "test_courier_login_wrong_password", "status": "passed", "description": "Система вернёт ошибку при неверном пароле.", "start": 1726599603072, "stop": 1726599603211, "uuid": "262e255c-57b6-419a-81a4-19867db68564", "historyId": "15a27e6e9549a4a5045654e236d34d1e", "testCaseId": "15a27e6e9549a4a5045654e236d34d1e", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_login_wrong_password", "labels": [{"name": "tag", "value": "courier_login"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "SC-WS-01209"}, {"name": "thread", "value": "20064-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}